### ISSUE TITLE
HingeJoint - Inspector does not allow negative target_velocity

### ISSUE THREAD

**Issue by xemjeff** [godotengine/godot#19203]
**Godot version:**
3.1.dev  (master branch, as of 5/25/18)

**OS/device including version:**
Windows 10

**Issue description:**
The inspector enforces values for the HingeJoint target_velocity > 0.  This is a problem, since we want to be able to reverse the direction of the motor without having to rotate the hinge joint 180 degrees.
You can set the velocity to negative values in code, which does reverse the direction of the motor:
```
hinge_joint.set_param(HingeJoint.PARAM_MOTOR_TARGET_VELOCITY,velocity)
```

**Steps to reproduce:**
Add HingeJoint node. Try to set the target_velocity in the inspector to a negative value. The inspector will set the value to 0.01.

**Comment by xemjeff** [comment]
I can fix this by changing the hint range at **physics_joint.cpp line#273**

`	ADD_PROPERTYI(PropertyInfo(Variant::REAL, "motor/target_velocity", PROPERTY_HINT_RANGE, "0.01,4096,0.01"), "set_param", "get_param", PARAM_MOTOR_TARGET_VELOCITY);`

I can change "0.01" to "-4096". But the value of 4096 seems really excessive.  The velocity is in radians/sec,  The max value of 4096 give us around 650 rev/sec or 39,000  RPM

I would use a range of  -200 to 200 which gives max motor speed of  (32 rev/sec). Consider this if you are running at 60 FPS.

@AndreaCatania @akien-mga 
Let me know your thoughts on changing the motor target velocity range as suggested. I will submit PR.

**Comment by akien-mga** [comment]
I'm not familiar with the feature, but yes I'm fine with reducing the min/max values as long as they still make sense for 99% of users.

If it still makes sense for some users to use values beyond the range specified, you can use the new `or_greater` and `or_lesser` hints like this: `"-200,200,0.01,or_greater,or_lesser". In 3.1+, this will limit the slider to [-200, 200] but let users input something beyond this range manually.

**Comment by akien-mga** [comment]
Fixed by #19246.
